diff -ru custodia-0.1.0.orig/custodia/message/kem.py custodia-0.1.0/custodia/message/kem.py
--- custodia-0.1.0.orig/custodia/message/kem.py	2016-09-07 16:30:48.275037634 +0200
+++ custodia-0.1.0/custodia/message/kem.py	2016-09-07 16:31:20.564570715 +0200
@@ -184,7 +184,7 @@
 
         ktype = self.client_keys[KEY_USAGE_ENC].key_type
         if ktype == 'RSA':
-            enc = ('RSA1_5', 'A256CBC-HS512')
+            enc = ('RSA-OAEP', 'A256CBC-HS512')
         else:
             raise ValueError("'%s' type not supported yet" % ktype)
 
@@ -397,7 +397,7 @@
                        JWK(**self.client_keys[KEY_USAGE_ENC])]
         cli = KEMClient(server_keys, client_keys)
         kem = KEMHandler({'KEMKeysStore': self.kk})
-        req = cli.make_request("key name", encalg=('RSA1_5', 'A256CBC-HS512'))
+        req = cli.make_request("key name", encalg=('RSA-OAEP', 'A256CBC-HS512'))
         kem.parse(req, "key name")
         msg = json_decode(kem.reply('key value'))
         rep = cli.parse_reply("key name", msg['value'])
